server/controllers/message.controller.js

```diff
@@ -174,10 +174,26 @@ function list(req, res) {
         queryObject.limit = req.query.limit;
     }
 
+    if (req.query.offset) {
+        queryObject.offset = req.query.offset;
+    }
+
     if (req.query.summary) {
         queryObject.attributes = ['subject', 'from', 'createdAt'];
     }
 
+    if (req.query.recieved) {
+        queryObject.where.to = { $contains: req.user.userName };
+    }
+
+    if (req.query.sent) {
+        queryObject.where.from = req.user.userName;
+    }
+
+    if (req.query.unread) {
+        queryObject.where.readAt = null;
```

Comment:
Because `readAt` was already `undefined` is this actually changing anything?

Reply:
This for sequelize. It not to unset an object its to query where the column's value is NULL.

https://www.garysieling.com/blog/sequelize-is-null-is-not-null

Reply:
Got it, just wanted to confirm that it translates correctly.